api/graphql/reservations/reservation_serializers.py

```diff
@@ -402,8 +402,8 @@ def validate(self, data):
     def validated_data(self):
         validated_data = super().validated_data
         validated_data["user"] = self.instance.user  # Do not change the user.
-        validated_data["confirmed_at"] = datetime.datetime.now(
-            tz=get_default_timezone()
+        validated_data["confirmed_at"] = datetime.datetime.now().astimezone(
```

Comment:
Time zone helpers might be more readable here, up to preference :smile: .

[now()](https://docs.djangoproject.com/en/3.2/ref/utils/#django.utils.timezone.now)
[localtime()](https://docs.djangoproject.com/en/4.0/ref/utils/#django.utils.timezone.localtime)

So: `localtime(now())`